ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -181,14 +181,10 @@ class _PprofConverter(object):
             ),
         )
 
-        nevents = len(events)
-        sampling_ratio_avg = sum(event.capture_pct for event in events) / nevents / 100.0
-        total_alloc = sum(event.nevents for event in events)
-        number_of_alloc = total_alloc * sampling_ratio_avg
-        average_alloc_size = sum(event.size for event in events) / float(nevents)
-
-        self._location_values[location_key]["alloc-samples"] = nevents
-        self._location_values[location_key]["alloc-space"] = round(number_of_alloc * average_alloc_size)
+        self._location_values[location_key]["alloc-samples"] = sum(event.nevents for event in events)
```

Comment:
❓ Shouldn't the number of events also be adjusted based on the `capture_pct`? Or is the total number of events, rather than the ones that got sampled?

Reply:
It looks like `event.nevents` is already counting the total number, as `capture_pct` is derived from that:
https://github.com/DataDog/dd-trace-py/blob/2f5304522c14c031067b190e81b79047cffc2d90/ddtrace/profiling/collector/memalloc.py#L144-L166